### PR TITLE
Fix Encounter Lists for views, add forms and make them launchable, add newer clinical views

### PR DESCRIPTION
**Pull request by ODORA0**

### CODE REVIEW

File: packages/esm-tb-app/src/pages/treatment/treatment-follow-up.component.tsx [hunk 1]
```diff
@@ -21,16 +21,16 @@ const TBTreatmentFollowUp: React.FC<PatientChartProps> = ({ patientUuid }) => {
         </TabList>
         <TabPanels>
           <TabPanel>
-            <DREnrollmentList patientUuid={patientUuid} />
+            <DSEnrollmentList patientUuid={patientUuid} />
```
[the remark this answers is not in the document]
Comment (ODORA0) [reply packages/esm-tb-app/src/pages/treatment/treatment-follow-up.component.tsx]: @semujju Done

File: packages/esm-tb-app/src/pages/treatment/treatment-follow-up.component.tsx [hunk 2]
```diff
@@ -21,16 +21,16 @@ const TBTreatmentFollowUp: React.FC<PatientChartProps> = ({ patientUuid }) => {
...
           </TabPanel>
           <TabPanel>
-            <DRFollowupList patientUuid={patientUuid} />
+            <DSFollowupList patientUuid={patientUuid} />
```
[the remark this answers is not in the document]
Comment (ODORA0) [reply packages/esm-tb-app/src/pages/treatment/treatment-follow-up.component.tsx]: @semujju Done

File: packages/esm-tb-app/src/pages/treatment/treatment-follow-up.component.tsx [hunk 3]
```diff
@@ -21,16 +21,16 @@ const TBTreatmentFollowUp: React.FC<PatientChartProps> = ({ patientUuid }) => {
...
           </TabPanel>
           <TabPanel>
-            <DSEnrollmentList patientUuid={patientUuid} />
+            <DREnrollmentList patientUuid={patientUuid} />
```
[the remark this answers is not in the document]
Comment (ODORA0) [reply packages/esm-tb-app/src/pages/treatment/treatment-follow-up.component.tsx]: @semujju Done

**Comment by Daphne210** [comment]
1. All the TB Treatment & Follow-up forms do not save.
2. Most dropdown selections within the TB Treatment & Follow-up forms are still blank.
3. The EID Follow-up form does not save.
4. Missing HIV Clinical Assessment form.
5. EID Summary Form does not save.
6. Still some skip logics missing.
7.  Contact Tracing form does not save.
8. IPD Palliative Form does not save.

**Comment by Daphne210** [comment]
Admission Form for IPD does not show up.

**Comment by ODORA0** [comment]
I think that is beyond the scope of the PR requirements. This PR was to tackle the following

1. Fix Encounter Lists for views
2. Add forms to the lists and make them launch-able
3. Add the newer clinical views that were discussed in the hackathon

**Comment by Daphne210** [comment]
> @Daphne210 I think that is beyond the scope of the PR requirements. This PR was to tackle the following
> 
> 1. Fix Encounter Lists for views
> 2. Add forms to the lists and make them launch-able
> 3. Add the newer clinical views that were discussed in the hackathon

@ODORA0 this has been noted. Though I tagged that to get a starting point for the forms team on the changes made.

**Review by Mwanje — APPROVED** [review]

**Pull request merged**